docs/technical_reference/flowsheets/membrane_distillation.rst

```diff
@@ -0,0 +1,106 @@
+   * `Heat Exchanger <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html>`_
+   * `Heater and chiller <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/heater.html>`_
+   * `Mixer <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/mixer.html>`_
+   * `Separator <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+
+Documentation for the property model can be found here:
+   * `Seawater Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_
+   * `Water Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
+
+This flowsheet aims to solve an optimization problem that minimizes the levelized cost of water (LCOW) with a specified overall water recovery.
+LCOW can be represented by the following equation where :math:`Q` represents product volumetric flow, :math:`f_{crf}` represents the capital recovery factor, :math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and :math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf} \cdot C_{cap,tot} + C_{op,tot}}{f_{util} \cdot Q}
+
+Degrees of Freedom
+------------------
+
+In addition to the overall water recovery, the following variables need to be specified based on the default settings:
+   * Feed water conditions (flow, temperature, pressure, component concentrations)
+   * Pump efficiency
+   * Membrane permeability coefficient, thickness, and thermal conductivity
+   * MD Hot and cold channel spacer porosity and height
+   * Heat exchanger overall heat transfer coefficient
+
+
+Flowsheet Specifications
+------------------------
+
+.. csv-table::
+   :header: "Description", "Units", "Value"
+
+   "**MD Module**"
+   "Membrane permeability coefficient", "1e-10", "m/Pa/s"
+   "Membrane thickness", "1e-4", "m"
+   "Membrane thermal conductivity", "0.2", "W/m/K"
+   "MD hot and cold channels height", "0.0019", "m"
+   "MD hot and cold channels spacer porosity", "0.77", "dimensionless"
+   "**Heat Exchanger**"
+   "Overall heat transfer coefficient", "2000", "W/m²K"
+   "**Pumps**"
+   "Efficiency", "0.8", "dimensionless"
+   "**Heater**"
+   "Maximum outlet temperature", "90", "°C"
+   "**Cooler**"
+   "Minimum outlet temperature", "10", "°C"
+
+
+
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recycle_ratio``, ":math:`\text{dimensionless}`"
+
+Additional Constraints
+----------------------
+
+There is an extra inequality constraint to ensure the Liquid Entry Pressure (LEP) to prevent membrane wetting:
```

Comment:
slightly confusing sentence... 

should it be something like "...to ensure the Liquid Entry Pressure (LEP) is sufficiently low to prevent membrane wetting"?

Reply:
Good catch. Thanks. modified to " There is an extra inequality constraint to ensure the hydraulic pressure inside the MD module is lower than the Liquid Entry Pressure (LEP) to prevent membrane wetting"